themes/spichka/resources/styles/app.scss

```diff
@@ -0,0 +1,96 @@
+$font-size-base: 1rem;
+$h2-font-size: $font-size-base * 2.5;
+$link-hover-color: #ff6f61;
+$text-color: #000;
+
+@import 'bootstrap/scss/bootstrap'; // TODO: leave only needed modules
+@import 'common/fonts';
+@import './archive';
+@import './plugins';
+@import './post-card';
+@import './single-post';
+@import './utils';
+@import './buttons';
+@import '~swiper/scss';
+@import '~swiper/scss/autoplay';
+@import '~swiper/scss/effect-fade';
+
+body {
+  font-family: Montserrat, sans-serif;
+  color: $text-color;
+}
+
+img {
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: contain;
```

Comment:
Not sure that it would look fine in all the occurances, as 'contain' shrinks image in original ratio if its too wide, or tall and it leaves blank spaces on the sides